CONTRIBUTING.md

```diff
@@ -48,15 +49,18 @@ Right now the most common use-case for syntax highlight using tree-sitter is
 [nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter), which
 means much of our testing is in relation to it. You can find the syntax
 highlighting tests in `test/highlight/*.scala`. You can read more about this
-type of testing
-[here](https://tree-sitter.github.io/tree-sitter/syntax-highlighting#unit-testing). These test will be automatically ran with `npm run test`.
+type of testing 
+[here](https://tree-sitter.github.io/tree-sitter/3-syntax-highlighting.html#unit-testing). 
+These test will be run automatically with `npm run test`.
```

Comment:
Is this still true? I only see `node --test bindings/node/*_test.js` in (package.json)[https://github.com/tree-sitter/tree-sitter-scala/blob/master/package.json#L50C14-L50C49]. CI seems like it has different actions for parser and binding tests.

Reply:
I think this is also `tree-sitter test`.